controller/array_action/array_mediator_ds8k.py

```diff
@@ -591,7 +592,9 @@ def delete_snapshot(self, snapshot_id):
             raise array_errors.SnapshotNameBelongsToVolumeError(api_volume.name,
                                                                 self.service_address)
         self._check_snapshot_use_status(snapshot_id, api_volume.flashcopy)
```

Comment:
consider renaming to something like `_ensure_snapshot_is_not_in_use`

Reply:
done